gradle.properties

```diff
@@ -1,5 +1,5 @@
 kotlin.code.style=official
 kotlin_version =1.6.21
-release_version=4.7.3
+release_version=4.8.0
```

Comment:
Let's bump maintenance version instead since we haven't added any additional functionality